packages/rn-tester-e2e/test/screenObjects/buttonComponent.screen.js

```diff
@@ -0,0 +1,67 @@
+import Utils from '../helpers/utils';
+
+class ButtonComponentScreen {
+
+    buttonScreenElement = Utils.platformSelect({
+        ios: '[label="Button"]',
```

Comment:
Can we search by test id instead of any string? Also means we can remove this per-platform bits.